compile_tools.sh

```diff
@@ -155,6 +155,17 @@ build_openh264() {
   cp testbin/layer2.cfg $TOOLDIR/layer2.cfg
 }
 
+build_libavc() {
+  # Build the Android M libavc
+  if [ ! -d libavc ]; then
+    git clone https://android.googlesource.com/platform/external/libavc
+  fi
+  # TODO(hta): Pick a version of libavc to compile. This compiles tip-of-tree.
+  (cd libavc; make -f ../../src/Makefile.libavcenclib)
```

Comment:
This looks weird. You cd one directory down, then you .. two directories up? Is that as it should be? I don't know which dir this starts from.